### DISCUSSION TITLE
Secret Scanning Alerts API - 404

### DISCUSSION THREAD

**Discussion by shahithyakumar** [community/community#60126] [Code Security]
### Select Topic Area

Question

### Body

Hi,

I'm trying to get the list of Secrets scanning alerts for my repository. I tried both curl command and the gh cli command given [here](https://docs.github.com/en/rest/secret-scanning/secret-scanning?apiVersion=2022-11-28#list-secret-scanning-alerts-for-a-repository). Provided correct token with all the required permissions. But it is throwing out 404.

Is it just for me, or for everyone?

**Comment by aashah** [comment]
Hey, sorry for the delay in responding. Is this still happening?

Some more details would be helpful, such as what repository you're doing this for (i.e. github.com/aashah/foobar), an example of the curl, a list of the scopes you've given the token, and maybe also a screenshot of the settings page to verify enablement.

**Reply by shahithyakumar** [reply]
Hey, sorry for the delay.
No worries, I've identified the mistake I was making when passing the curl command. It's working fine now.

Thanks! 😄
